invenio_github/api.py

```diff
@@ -371,6 +371,11 @@ def related_identifiers(self):
         )
 
     @cached_property
+    def version(self):
+        """Extract the vesion from the release tag."""
+        return self.release.get('tag_name', '')
```

Comment:
Maybe just `self.release.get('tag_name')`, since `None` value is a bit cleaner I guess if there was no tag for some weird reason